layers/+lang/c-c++/README.org

```diff
@@ -244,9 +244,9 @@ Additionally, if you have =c-c++-enable-google-newline= variable set then
 set that up like this:
 
 #+BEGIN_SRC emacs-lisp
-  (c-c++ :variables
-         c-c++-enable-google-style t
-         c-c++-enable-google-newline t)
+ (c-c++ :variables
+ c-c++-enable-google-style t
+ c-c++-enable-google-newline t)
```

Comment:
Hmm... this looks wrong. I would have expected it to suggest removing only one space from the beginning of all 3 lines. @JAremko?

Reply:
@sdwolfz  That's because someone forgot `#+END_SRC` :shipit:    

Don't think it is autofixable :thinking:

Reply:
Dammit, I was specifically looking for this last night but missed it multiple times. For some reason that block if code did not stand out to me.